vignettes/adrg.qmd

```diff
@@ -0,0 +1,785 @@
+| ADAE - Adverse Events Analysis Dataset       | OCCURRENCE DATA STRUCTURE      |          | x      |                         |           | One record per subject per adverse event                                      |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+|                                              |                                |          |        |                         |           |                                                                               |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+| ADLBC - Analysis Dataset Lab Blood Chemistry | BASIC DATA STRUCTURE           |          | x      |                         |           | One or more records per subject per analysis parameter per analysis timepoint |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+|                                              |                                |          |        |                         |           |                                                                               |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+| ADTTE - AE Time To 1st Derm. Event Analysis  | BASIC DATA STRUCTURE           | x        | x      |                         |           | One or more records per subject per analysis parameter per analysis timepoint |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+
+: {tbl-colwidths="\[20,20,10,10,20,10,10\]"}
+
+### ADSL - Subject-Level Analysis Dataset
+
+The subject level analysis dataset (ADSL) contains required variables for demographics, treatment groups, and population flags. In addition, it contains other baseline characteristics that were used in both safety and efficacy analyses. All patients in DM were included in ADSL. The following are the key population flags are used in analyses for patients: • SAFFL – Safety Population Flag (all patients having received any study treatment) • ITTFL – Intent-to-Treat Population Flag (all randomized patients)
```

Comment:
were the SAFFL and ITTFL meant to be a bulleted list?

Reply:
Thanks for catching this.  Fixed.